src/screens/Topics/index.js

```diff
@@ -214,7 +216,7 @@ const styles = StyleSheet.create({
     fontWeight: '600',
     fontSize: normalizeFontSize(36),
     lineHeight: normalizeFontSize(43.57),
-    color: '#11243D',
+    color: colors.bunting,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing hardcoded color values with constants. This improves maintainability and consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->